product_lot_sequence/models/product.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    sequence_id = fields.Many2one(
```

Comment:
Please change to 'lot_sequence_id'.

Same for all fields under. Your field names are too generic

Reply:
Done :+1: